### DISCUSSION TITLE
Have the same resolution as the original file, .NET

### DISCUSSION THREAD

**Discussion by valdomero777** [ImageMagick/ImageMagick#4425] [Help]
I am working with postscripts files. I am converting those files into images, but when I do this process the resolution of the image is lower than the PostScript file. That image I want to print in a thermal printer. If I send to print only the PS the resolution is better, and I want that resolution
Exists one way to have the same resolution? I am working with .net

Thanks in advace

**Comment by fmw42** [comment]
Use supersampling. Specify a high density when reading the file and then resize back to the original size.

`convert -density 288 image.ps -resize 25% image.png`

Nominal PS density is 72. So 72*4=288 is four times normal. So resize by 1/4=25%

**Reply by valdomero777** [reply]
thanks! I forgot say I am working with .net
how could be?

**Reply by fmw42** [reply]
Sorry, I do not know Magick.Net.  Someone else will have to convert my command line code to .Net
